server/index.ts

```diff
@@ -0,0 +1,30 @@
+import mongoose from "mongoose";
+import config from "config";
+
+export default async () => {
+  if (mongoose.connections[0].readyState) return;
+  // const DB_URL: string = config.db.url || "";
+  // 
+  // await mongoose
+  //   .connect(DB_URL, {
+  //     ...config.db.options,
+  //     dbName: config.db.name
+  //   })
+  //   .catch(error => {
+  //     console.error("Database connection failed.");
+  //     console.error(" > " + error);
+
+  //     throw error;
+  //   });
+
+    await mongoose
+    .connect("mongodb+srv://kfidan:ZiPwdaph0GnqjlbF@cluster0.wzdut.mongodb.net/hack4impact?retryWrites=true&w=majority", {
```

Comment:
Not sure how secure it is to leave this visible in the repo, but I think it contains the password to the mongo cluster in the connection string, so we might need to store this in an environment variable.

Reply:
You're right. This one connects to a dummy collection under my personal acc but i'll change it before the merge